lib/jxl/enc_frame.cc

```diff
@@ -1134,29 +1129,30 @@ Status EncodeGlobalDCInfo(const PassesSharedState& shared, BitWriter* writer,
 }
 
 Status EncodeGlobalACInfo(PassesEncoderState* enc_state, BitWriter* writer,
-                          ModularFrameEncoder* modular_frame_encoder,
-                          AuxOut* aux_out) {
+                          ModularFrameEncoder* enc_modular, AuxOut* aux_out) {
   PassesSharedState& shared = enc_state->shared;
-  JXL_RETURN_IF_ERROR(DequantMatricesEncode(
-      shared.matrices, writer, kLayerQuant, aux_out, modular_frame_encoder));
+  JXL_RETURN_IF_ERROR(DequantMatricesEncode(shared.matrices, writer,
+                                            kLayerQuant, aux_out, enc_modular));
   size_t num_histo_bits = CeilLog2Nonzero(shared.frame_dim.num_groups);
-  if (num_histo_bits != 0) {
+  if (!enc_state->streaming_mode && num_histo_bits != 0) {
```

Comment:
I assume these get written somewhere else in this case? If so, a comment would be useful for us to remember in the future.

Reply:
I added a comment to the function that explains this.